### PR TITLE
.coafile: Set max_line_length

### PR DESCRIPTION
**Pull request by ankitxjoshi**
Fixes https://github.com/coala/cEPs/issues/134

### CODE REVIEW

**Comment by ankitxjoshi** [comment]
There are around 50 violations in this repo.

**Comment by jayvdb** [comment]
Please fix them?

File: cEP-0014.md [hunk 1]
```diff
@@ -851,7 +862,7 @@ def get_bears_with_matching_dependencies(
             # Check if names of requirements match
             if dep.value == req:
                 if (are_compatible_versions(dep.version, req_info["version"])
-                        and are_compatible_req_types(dep.type, req_info["type"])):
+                    and are_compatible_req_types(dep.type, req_info["type"])):
```
Comment (jayvdb) [inline cEP-0014.md]: wow, this one is wrong.  that is inappropriate indentation for python

File: .coafile [hunk 2]
```diff
@@ -6,6 +6,7 @@ bears = SpaceConsistencyBear
 [markdown]
 files = **.md
 bears = MarkdownBear
+max_line_length = 79
```
Comment (jayvdb) [inline .coafile]: can we set this to 80, and that should mean there are less changes in the PR?

**Review by jayvdb — CHANGES_REQUESTED** [review]

File: .coafile (hunk 2 above)
Comment (ankitxjoshi) [reply .coafile]: We can. But do I have to reset everything? That's gonna be pain.

File: cEP-0014.md (hunk 1 above)
Comment (ankitxjoshi) [reply cEP-0014.md]: The variable name is too big to accommodate. What should be the alternative?

File: .coafile (hunk 2 above)
Comment (jayvdb) [reply .coafile]: Yes. Objective is minimal change of line attribution

Surya already did change lines using 80 cpl. Annoying to have another large set if changes without good reason.

File: cEP-0014.md (hunk 1 above)
Comment (jayvdb) [reply cEP-0014.md]: change variable name?

Comment (jayvdb) [reply cEP-0014.md]: Or use inline ignore

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
ack https://github.com/coala/cEPs/pull/157/commits/87ae1ac46c45cac007d450b086c4134b30d0ff64

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**